articles/quickstart/spa/_includes/_calling_api_protect_resources.md

```diff
@@ -1,7 +1,12 @@
 ## How Do I Protect my API Resources?
 
-To restrict access to the resources served by your API, a check needs to be made to determine whether the incoming request contains valid authorization information. When you implement Auth0 in your client-side application and data API, that authorization information is the `access_token` issued for your user. This token must be checked against the [JSON Web Key Set (JWKS)](https://auth0.com/blog/navigating-rs256-and-jwks/) for your Auth0 account to verify that it is valid.
+To restrict access to the resources served by your API, check the incoming requests for valid authorization information. 
+The authorization information is in the `access_token` created for the user. To see if the token is valid, check it against the [JSON Web Key Set (JWKS)](https://auth0.com/blog/navigating-rs256-and-jwks/) for your Auth0 account.
```

Comment:
we also have a separate doc for validating access tokens, I'd link to that https://auth0.com/docs/api-auth/tutorials/verify-access-token